tools/klee/main.cpp

```diff
@@ -1587,6 +1652,44 @@ int main(int argc, char **argv, char **envp) {
         << handler->getNumPathsExplored() << ", among which\n";
   stats << "KLEE: done:     early-terminating paths (instruction time limit, solver timeout, max-depth reached) = "
         << handler->getEarlyTermination() << "\n";
+
+  if (INTERPOLATION_ENABLED) {
+    stats << "KLEE: done:     average branching depth of completed paths = "
+          << (double)(handler->getTotalBranchingDepthOnExitTermination() +
+                      handler->getTotalBranchingDepthOnEarlyTermination() +
+                      handler->getTotalBranchingDepthOnErrorTermination()) /
+                 (double)(handler->getExitTermination() +
+                          handler->getEarlyTermination() +
+                          handler->getErrorTermination()) << "\n";
+    if (handler->getSubsumptionTermination() == 0.0) {
+      stats << "KLEE: done:     average branching depth of subsumption paths = "
```

Comment:
As with my previous comments for other locations, I think "subsumed paths" is better than "subsumption paths" in the message string.